client/analytics/report/customers/config.js

```diff
@@ -167,6 +167,81 @@ export const advancedFilters = {
 				} ) ),
 			},
 		},
+		order_count: {
+			labels: {
+				add: 'No. of Orders',
```

Comment:
You'll need translations for all these strings

Reply:
🤦‍♂️ Thanks for catching!  Added.